cardano-cli/src/Cardano/CLI/Read.hs

```diff
@@ -633,6 +633,12 @@ readCddlTx = readFileOrPipeTextEnvelopeCddlAnyOf teTypes
     , FromCDDLTx "Unwitnessed Tx AlonzoEra" CddlTx
     , FromCDDLTx "Unwitnessed Tx BabbageEra" CddlTx
     , FromCDDLTx "Unwitnessed Tx ConwayEra" CddlTx
+    , FromCDDLTx "TxSignedShelley" CddlTx
```

Comment:
We ought to have a property test that catches when this isn't updated.

Reply:
Also we should define this based on the `HasTextEnvelope` instance for `Tx era`. That would stop us from having to manually update this each time a new era is introduced.

Reply:
Addressed now